pkg/client/k8s.go

```diff
@@ -72,10 +73,33 @@ type K8sClient struct {
 	CLIENT CREATION METHODS AND RELATED OPERATIONS
 */
 
-// NewK8sClient creates a new K8sClient
+// A wrapper around newK8sClient to handle errors in creating
```

Comment:
The comment should begin like `NewK8sClient ...`

This is a golint error.

Reply:
Done